PD-1560 / 25.04 / Update SMARTTestsSCALE.md (#3362)
truenas/documentation: content/SCALE/SCALETutorials/DataProtection/SMARTTestsSCALE.md

Next, select the test type from the **Type** dropdown and then click **Start**.

> Following S.M.A.R.T. test initiation, a loading bar appears next to the chosen disks. You can refer to this loading bar to check the status of your running tests, as it updates in accordance with test completion rates. If your system has disks that cannot support S.M.A.R.T. tests, the dialog window lists those beneath the loading bar.

Reviewer: Suggestions to make this content more concise. How about this? Also, in the last sentence, what does those refer to? the disks?
```suggestion
After starting a S.M.A.R.T. test a loading bar shows next to the chosen disks.
Refer to this loading bar for the status of your running tests, as it updates in accordance with test completion rates.
If your disks that cannot support S.M.A.R.T. tests, the dialog window lists those beneath the loading bar.
```
